### DISCUSSION TITLE
Extracting table with no vertical lines (only horizontal lines) doesn't work

### DISCUSSION THREAD

**Discussion by jjjhill** [jsvine/pdfplumber#1104] [Ask for help with specific PDFs]
## Describe the bug

extract_table with a table with no vertical lines fails to recognize the entire table. It cuts off some of the right side.

## Have you tried [repairing](https://github.com/jsvine/pdfplumber/blob/stable/docs/repairing.md) the PDF?
Yes

## Code to reproduce the problem
```python
table_settings = {
    "vertical_strategy": "text",
    "horizontal_strategy": "lines",
}
left = p.crop((0, 0, p.bbox[2]/2, p.bbox[3]))

img = left.to_image(resolution=400)
img.debug_tablefinder(table_settings)
```

## PDF file

[example3.pdf](https://github.com/jsvine/pdfplumber/files/14332159/example3.pdf)

## Expected behavior

Entire table was captured.

## Actual behavior

Partial table was captured.

## Screenshots

Here is the result of `debug_tablefinder`:
![debug](https://github.com/jsvine/pdfplumber/assets/16979310/b16b46da-6b82-4e55-9253-9c0e181cf51f)

NOTE: I've also tried cropping the image strictly to include the table only. Same result.
## Environment

- pdfplumber version: 0.10.4
- Python version: 3.10.6
- OS: Windows

## Additional context

I've tried all relevant parameters `table_settings` with no luck.
Any suggestions?

**Comment by jsvine** [comment]
Hi @jjjhill, and thanks for your interest in `pdfplumber`.  The `text` strategy is, unfortunately, a bit finicky and layout-dependent. But I don't see an actual bug at play here, so I'm moving this to a discussion. 

For this particular PDF, I'd recommend using the `"explicit_vertical_lines": [...]` setting, and then using some logic to determine what those lines should be. One common strategy is to examine the *horizontal* lines/rects, and using their `x0/x1` positioning as input. You might also use the the text in the header to identify the `x` position that divides the two columns.
